plextraktsync/commands/inspect.py

```diff
@@ -37,8 +38,13 @@ def inspect_media(id: str):
 
     media = pm.item
     print(f"Title: {media.title}")
-    if media.type == 'movie' and media.editionTitle:
-        print(f"Edition Title: {media.editionTitle}")
+    if media.type == 'movie':
+        try:
+            edition = media.editionTitle
+        except (ReadTimeout, RequestException):
+            edition = None
+        if edition:
+            print(f"Edition Title: {edition}")
```

Comment:
add method to PlexLibraryItem and use that instead of doing this in inspect directly on pm.item